src/Compilers/Core/Portable/Syntax/SyntaxTokenList.cs

```diff
@@ -77,16 +74,21 @@ public static SyntaxTokenList Create(ReadOnlySpan<SyntaxToken> tokens)
             if (tokens == null)
```

Comment:
Same here

Reply:
In this case, I think we can delete this code path and hit `case 0` in the below switch, getting equivalent behavior. Not blocking if the change doesn't feel right to you.

Reply:
sure!